src/logic/map_objects/bob.cc

```diff
@@ -1052,8 +1056,9 @@ void Bob::Loader::load(FileRead& fr) {
 				bob.anim_ = bob.descr().get_animation(animname, &bob);
 			} else {
 				bob.anim_ = bob.descr().main_animation();
-				log("Unknown animation '%s' for bob '%s', using main animation instead.\n",
-				    animname.c_str(), bob.descr().name().c_str());
+				log_warn_time(egbase().get_gametime(),
```

Comment:
```suggestion
				log_warn(
```

We don't care about game time during game loading. Check the remaining loader functions too.